src/project-css/app/Console/Commands/exportTable.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class exportTable extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'table:export {tablename : name of table} 
+                                         {--field=* : requested fields of table}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Table';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        if (!Schema::hasTable($this->argument('tablename'))) { 
+            return $this->error('Table Doesn\'t Exist.');
+        }
+
+        // allowed fields
+        $allowed = $this->option('field');        
+
+        $file = fopen('./storage/app/exports/' . $this->argument('tablename') .'.csv', 'w');
+        fputcsv($file, $allowed);
+        $records = DB::table($this->argument('tablename'))->get();
+
+        foreach ($records as $record) {
```

Comment:
You are making a CSV here?  Its a little confusing to read with the array filtering happening here, that might be a good place to offset that logic to another function instead of using an anonymous callback function.